assets/js/index.js

```diff
@@ -0,0 +1,12 @@
+// Mobile Nav Toggle
+const mobileBtn = document.querySelector('.mobile-toggle-btn');
+const headerNav = document.querySelector('.header');
+const navLink = document.querySelector('.link-nav');
+
+mobileBtn.addEventListener('click', () => {
+  headerNav.classList.toggle('active');
+});
+
+navLink.addEventListener('click', () => {
+  headerNav.classList.remove('active');
+});
```

Comment:
- [ ] Great job with your project 👍  please make sure when the user clicks (or taps) any of the mobile menu options, the mobile menu disappears as stated in the project requirements 🙏 , which is also best practice ☺️ .